includes/404-widget.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * LSX functions and definitions - Config.
+ *
+ * @package    lsx
+ * @subpackage 404-widget
+ */
+
+function lsx_widget_area_404_init() {
+	register_sidebar( array(
+			'name'          => esc_html__( '404 page', 'lsx' ),
+			'id'            => 'sidebar-404',
+			'before_widget' => '<aside id="%1$s" class="widget %2$s">',
+			'after_widget'  => '</aside>',
+			'before_title'  => '<h3 class="widget-title">',
+			'after_title'   => '</h3>',
+		) );
+	}
+
+add_action( 'widgets_init', 'lsx_widget_area_404_init' );
+
+function my_search_placeholder() {
```

Comment:
@viscosho please check if the function exists before to create it. Check the other include files to see how we do it in LSX core theme.